en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -1,96 +1,96 @@
-### Step 2: Drill into the analytics
+    To do this, click on the relevant latency value. In this example, the status code is `500`, which means the request resulted in an error state.
 
-![Diagnostic view](../assets/img/observability/diagnostics-view.png){.cInlineImage-full}
 
-Follow the steps below to learn how to drill down and confirm the root cause for the anomalies we detected in the **Latency and Throughput** graphs.
+### Analyze diagnostic data
 
-1. Click on the **Diagnostics View** tab on the left panel. 
+The **Diagnostic View** allows you to further drill down the observability data you viewed in the **Throughput & Latency** tab to identify possible root causes for anomalies of a service.
 
-    !!!info
-        The time range selected for the **Throughput & Latency** graphs is applied to the **Diagnostics View** by default. We recommend capturing the time range you want to analyze in detail from the **Throughput & Latency** graph and then navigate to the **Diagnostics View**.
+The following image is an example of how you can view diagnostic data once you click on the **Diagnostics View** tab.
+ 
+![Diagnostic data](../assets/img/observability/diagnostics-view.png){.cInlineImage-full}
+
+!!!info
+    The time range applied to the **Diagnostics View** by default is the same time range for which you viewed the **Throughput & Latency** before clicking on this tab. 
         
-2. In the sample data, you can observe a couple of prominent errors appearing in the four bins.
-3. Select a bin that shows a higher occurrence of the error log "error while connecting to the hr-service”. Notice that this bin may include other errors which occurred at a lesser frequency.
+If you hold the pointer over a section where you see fluctuations in multiple bins, you can view the related logs.
+
+![Fluctuations in diagnostic data](../assets/img/observability/second-bin.png){.cInlineImage-full}
 
-    ![Diagnostic view - second bin](../assets/img/observability/second-bin.png){.cInlineImage-bordered}
+In this example, the logs indicate a higher occurrence of `error while connecting to the hr-service`. Although there are other errors also displayed, those error counts are comparatively much lower.
 
-4. Furthermore, when you analyze the graph, you can find a peak with a larger number of errors error. At this same time notice instances where you can see an increase in the throughput and a drop in latency. 
-5. By analyzing the graphs as above, you can confirm that the cause for the latency drop and the throughput spike is the connectivity issue displayed by the error logs.
-6. Observe that the error graph is also fluctuating over time as the service logs errors.
-7.  You can observe similar behavior in the other bins as well.
+In the same example, if you further analyze the sample diagnostic data, you can identify the connectivity error as the most common error.  In such error timestamps, you can also see an increase in the throughput and a drop in latency. 
```

Comment:
Actually, in this case, this is because of an error the code doesn't do some of the processing (simulated) and this results in a decrease in latency and an increase in overall throughput. We show the overall throughput (including errors).

But I think we should not mention about latency and throughput here as we are looking at the connectivity issue.